plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -274,8 +327,33 @@ global proc mayaUsd_updateSelectionKindMenuItem() {
             global string $gMayaUsdSelectModeSubMenu;
             $menuItems = `menu -q -itemArray $gMayaUsdSelectModeSubMenu`;
             for($i = 0; $i < size($menuItems); $i++) {
-                $label = `menuItem -q -label $menuItems[$i]`;
-                if($label == $selectionKind) {
+                string $label = `menuItem -q -label $menuItems[$i]`;
+                if(`tolower($label)` == $selectionKind) {
+                    menuItem -e -radioButton on $menuItems[$i];
+                    break;
+                }
+            }
+    }
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// mayaUsd_updatePointInstancesPickModeMenuItem
+// Update the menu item radio buttons in the Point Instances Pick Mode menu
+global proc mayaUsd_updatePointInstancesPickModeMenuItem() {
```

Comment:
New proc to update the point instances pick mode menu items. Very similar to the `SelectionKindMenuItem` just above. Also use all lowercase comparison.